src/constants.js

```diff
@@ -24,12 +24,18 @@ export const SAMPLED_MASK = 0x1;
 // DEBUG_MASK is the bit mask indicationg that a span has been marked for debug.
 export const DEBUG_MASK = 0x2;
 
+// DEFERRED_SAMPLING_MASK is the bit mask indicating that a span has been marked for deferred sampling.
```

Comment:
Well, that statement is akin to saying that the network isn't reliable, and thus the feedback loop might not be reliable as well. It does not claim that we cannot have a feedback loop.